consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/BlockTimer.java

```diff
@@ -24,15 +24,53 @@
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /** Class for starting and keeping organised block timers */
 public class BlockTimer {
 
+  private static final Logger LOG = LoggerFactory.getLogger(BlockTimer.class);
+
   private final ForksSchedule<? extends BftConfigOptions> forksSchedule;
   private final BftExecutors bftExecutors;
   private Optional<ScheduledFuture<?>> currentTimerTask;
   private final BftEventQueue queue;
   private final Clock clock;
 
+  // For testing only, it can be useful to set BFT block periods less than a second. This allows
```

Comment:
I know this is only for testing, but I still don't like the environment variable parsing in the main code here. Adding an experimental config option to QbftConfig would be much nicer. It's quite easy to pass the value into this class too since we create this in the QbftBesuControllerBuilder.